GVRf/Framework/framework/src/main/jni/gl/gl_texture.h

```diff
@@ -103,10 +103,15 @@ class GLTexture : public GLPendingTask {
             glGenTextures(1, &id_);
             glBindTexture(target_, id_);
 
+            GLfloat anisotropic_level;
+
+            glGetFloatv(GL_MAX_TEXTURE_MAX_ANISOTROPY_EXT, &anisotropic_level);
```

Comment:
I believe before making this call you need to be certain EXT_texture_filter_anisotropic is supported.